app/plugins/adp/src/components/DeliveryProject/CreateDeliveryProject.tsx

```diff
@@ -39,6 +43,11 @@ const CreateDeliveryProject: React.FC<CreateDeliveryProjectProps> = ({
     discoveryApi,
     fetchApi,
   );
+
+  const { allowed } = usePermission({
```

Comment:
Same as above

Reply:
allowed is a type on AsyncPermissionResult the backstage policy that sets up permissions